iOS/Resources/bin/arm64-apple-ios-clang

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
```

Comment:
Sorry, I didn't notice these earlier: all of the stubs should just use a shebang line of `/bin/sh` rather than `bash` since `bash` is officially deprecated on macOS, and there are no bash-isms used.